client/src/plugins/index.js

```diff
@@ -9,21 +9,23 @@
  */
 
 import CamundaPlugin from './camunda-plugin';
+import ElementTemplatesModal from './element-templates-modal';
+import ErrorTracking from './error-tracking';
 import PrivacyPreferences from './privacy-preferences';
+import ReportFeedback from './report-feedback';
 import UpdateChecks from './update-checks';
-import ErrorTracking from './error-tracking';
 import UsageStatistics from './usage-statistics';
-import ElementTemplatesModal from './element-templates-modal';
-import ZeebePlugin from './zeebe-plugin';
 import VersionInfo from './version-info';
+import ZeebePlugin from './zeebe-plugin';
 
 export default [
   CamundaPlugin,
+  ElementTemplatesModal,
```

Comment:
I'm wondering if we should start to add eslint rules for this, if we sort variables often.

I've seen more and more fellows on our team doing it (sorting things).

Reply:
Won't do it, as discussed.